task/sast-coverity-check/0.3/MIGRATION.md

```diff
@@ -0,0 +1,9 @@
+# Migration from 0.2 to 0.3
+
+- The required `IMAGE_DIGEST` parameter has been added back.
+
+## Action from users
+
+- All resources used task `sast-coverity-check` should be directed to use new `0.3` version.
+- The `IMAGE_DIGEST` parameter definition is required to be added for this task in the build pipeline.
```

Comment:
Does it make sense to [create a migration](https://github.com/konflux-ci/build-definitions?tab=readme-ov-file#task-migration) that would do it automatically?

Reply:
Fixed and script is added and approved

Reply:
Should we also add the script to the updated task folders as well, especially for the required tasks?